### PR TITLE
Fixes in pycompwa and improvements in expert system

### PR DESCRIPTION
**Pull request by spflueger**

### CODE REVIEW

File: pycompwa/CMakeLists.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ if(${Python3_FOUND})
 
   # Copy to build directory
   configure_file(${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in ${SETUP_PY})
-  add_custom_command(OUTPUT ${OUTPUT}
+  add_custom_target(pycompwa ALL
```
Comment (hansgans) [inline pycompwa/CMakeLists.txt]: This is not the same as the previous implementation. Please revert.

File: pycompwa/pycompwa/plotting/__init__.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import matplotlib.pyplot as plt
-# from matplotlib import rc
-from matplotlib.colors import LinearSegmentedColormap
+
```
Comment (hansgans) [inline pycompwa/pycompwa/plotting/__init__.py]: I would like to avoid modules containing a single file

Comment (spflueger) [reply pycompwa/pycompwa/plotting/__init__.py]: I forgot to add the `rootplotdatareader.py` file, I will update the pull request shortly

File: pycompwa/CMakeLists.txt (hunk 1 above)
Comment (spflueger) [reply pycompwa/CMakeLists.txt]: Hmm, I had some problems of python files being copied when I made modifications. I will check that again

Comment (spflueger) [reply pycompwa/CMakeLists.txt]: Yeah, I just tried it again. When I make some modifications to some python code (after the build), and I build again, the modifications are not copied to the build directory. Why do you need this `add_custom_command`?

**Comment by spflueger** [comment]
Ok, now everything should be included. If you are happy with the pull request you can merge it.

**Pull request merged**